public/app/features/alerting/unified/components/receivers/form/fields/TemplateSelector.tsx

```diff
@@ -303,7 +304,21 @@ function OptionCustomfield({
       <TextArea
         id={id}
         label={t('alerting.option-customfield.label-custom-template', 'Custom template')}
-        placeholder={option.placeholder}
+        placeholder={
+          option.placeholder
+            ? unescape(
```

Comment:
why do you need this unescape here?

Reply:
@soniaAguilarPeiron 

In this case, `currentTemplate` displays the value of `option.placeholder`, which comes from the backend. 

Here’s what it looks like without applying `unescape`:

<img width="615" height="155" alt="Screenshot 2025-09-15 at 17 26 40" src="https://github.com/user-attachments/assets/83932a9f-94c3-454b-95b5-b75ba5b75ebd" />

The text in an "escaped" form.